flatpak_builder_lint/checks/desktop.py

```diff
@@ -2,30 +2,14 @@
 import re
 import subprocess
 import tempfile
-from typing import Optional, Set
+
+from gi.repository import GLib
```

Comment:
do we need to add gi-python (or whatever it's called)?

It's only on the GNOME runtime.

Reply:
PyGobject? It's in the baseapp already, we use it already for the flatpak metadata parser in linter and flat manager.